README.md

```diff
@@ -2,29 +2,42 @@
 
 # PostCSS CSS Variables
 
-[PostCSS](https://github.com/postcss/postcss) plugin to transform [`CSS Custom Properties(CSS variables)`](http://dev.w3.org/csswg/css-variables/) syntax into a static representation. This plugin provides a future-proof way of using **most** of CSS variables features.
-
-CSS variables or CSS Custom Properties limited subset polyfill/shim.
-
-We strive for the most complete transformation but we/no plugin can achieve true complete parity according to the [specification](http://dev.w3.org/csswg/css-variables/) because of the DOM cascade unknowns.
+```
+npm install postcss-css-variables --save-dev
+```
 
-### [Changelog](https://github.com/MadLittleMods/postcss-css-variables/blob/master/CHANGELOG.md)
+[PostCSS](https://github.com/postcss/postcss) plugin to transform [`CSS Custom Properties (CSS variables)`](http://dev.w3.org/csswg/css-variables/) syntax into a static representation. This plugin provides a future-proof way of using **most** of CSS variables features, including selector cascading with some caveats, because this can only see the CSS, not the potentially dynamic HTML and DOM the CSS is applied to.
 
-### Install
+### Table of Contents
 
-`npm install postcss-css-variables --save-dev`
+ - [Changelog](https://github.com/MadLittleMods/postcss-css-variables/blob/master/CHANGELOG.md)
+ - [Code Playground](#code-playground)
+ - [Usage](#usage)
+ - [Syntax](#syntax)
+	 - [Defining Custom Properties: the `--*` family of properties](#defining-custom-properties-the----family-of-properties)
+	 - [Using Cascading Variables: the `var()` notation](#using-cascading-variables-the-var-notation)
+ - [Features](#features)
+	 - [Nested Rules](#nested-rules)
+ - [Why?](#why)
+	 - [Interoperability](#interoperability)
+	 - [Differences from `postcss-custom-properties`](#differences-from-postcss-custom-properties)
+ - [Caveats](#caveats)
+ - [Options](#options)
+ - [Quick Reference/Notes](#quick-referencenotes)
+ - [Testing](#testing)
 
-# [Code Playground](https://madlittlemods.github.io/postcss-css-variables/playground/)
+# Code Playground
```

Comment:
IMHO it looks very odd as a blue header. There's already a link in the body within the first sentence, anyways, so at least one of the links seems redundant.